rubix-tests/src/test/java/com/qubole/rubix/tests/TestCachingInputStream.java

```diff
@@ -26,6 +26,7 @@
 import org.apache.commons.logging.LogFactory;
 import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.fs.FSDataInputStream;
+import org.apache.hadoop.fs.FileSystem;
```

Comment:
Are the changes in both the test files mainly to make the current tests to work ? Am I correct to assume that there are no new unit tests yet for this feature ?

Reply:
Yes.. I have not added any new test cases for this feature.